manifests/cf-manifest/manifest/000-base-cf-deployment.yml

```diff
@@ -1033,23 +988,20 @@ instance_groups:
               restart_if_consistently_above_mb: ~
               restart_if_above_mb: ~
 
-          external_protocol: ~
-
-          tls_port: 9023
           mutual_tls: *cloud_controller_ng_mutual_tls
 
         ccdb: *cloud_controller_ng_ccdb
 
         uaa:
           ca_cert: ((certs_bosh_ca_cert))
-          ssl:
-            port: 8443
-          port: 8080
           clients:
             cc-service-dashboards:
               secret: ((secrets_uaa_clients_cc_service_dashboards_password))
             cc_routing:
               secret: ((secrets_uaa_cc_routing_secret))
+          ssl:
```

Comment:
uaa.ssl.port can be removed

Reply:
This was left in on purpose due to consistency with upstream: https://github.com/cloudfoundry/cf-deployment/blob/v1.14.0/cf-deployment.yml#L971-L972 (upstream's inconsistency :) )